doc/template.html

```diff
@@ -13,7 +13,7 @@
     <div id="column2" class="interior">
       <div id="intro" class="interior">
         <a href="/" title="Go back to the home page">
-          Node.js (1)
+          Node.js
```

Comment:
nm. wasn't paying attention to what file this was located in. our man page should be `NODE(1)`, but this looks good.